src/main.cpp

```diff
@@ -1160,7 +1160,8 @@ int main(int argc, char **argv) {
 
                     args.pop();
                     Termination term;
-                    os_spawn_process(exec_path, args, &term);
+                    // use NULL for exe, it's already included in args
```

Comment:
What do you think about making the API in os.cpp not even have the exe parameter? It would require the first arg to be the exe. That's what we ended up doing in the zig standard library and I think it's been a reasonable abstraction. That's how I was planning to fix this issue in the stage1 compiler.

Reply:
Sounds reasonable.  I'll see if I can update the PR tonight.

Reply:
I've updated the PR, let me know if you'd like any more changes.